### ISSUE TITLE
problem regarding to paging file

### ISSUE THREAD

**Issue by Eomtaesan** [Microsoft/CNTK#2319]
My total training data is about ~23GB, and I have 48GB RAM.
Then I think cntk can load the data to RAM, but MSI afterburner shows this.

![image](https://user-images.githubusercontent.com/22387152/30197356-2dd2411c-94a2-11e7-9962-eaa9a7f692cc.png)

This figure, paging file usage is shown when I am training, and this is my window setting & state

![image](https://user-images.githubusercontent.com/22387152/30197396-7e7f111c-94a2-11e7-8fbb-494955d3ace7.png)

![image](https://user-images.githubusercontent.com/22387152/30197316-e2db1256-94a1-11e7-8a6b-92928fc0e555.png)

There are some Korean, but I think you can understand the setting about paging file.
So, How to load totally the data to RAM?

**Comment by n17s** [comment]
You have to specify a large value for randomization_window_in_chunks in your minibatchSource. The size of a chunk is defined to be 32MB so if you specify randomization_window_in_chunks=800 that's 25GB so it should fit your file (this is assuming that 1 byte in disk will take 1 byte in RAM which might not be true for reasons such as working with compressed image data).

**Comment by Eomtaesan** [comment]
Thank your for the answer.
I will try it.
Can I set the option in brainscript?
I will try to write "randomization_window_in_chunks = 800" at the top of the cntk config file with brainscript

**Comment by n17s** [comment]
I think in brainscript there's a RandomizationWindow and the meaning is "samples", not chunks.

**Comment by Eomtaesan** [comment]
@n17s 
I try to find the option, "RandomizationWindow".
(https://docs.microsoft.com/en-us/cognitive-toolkit/brainscript-cntkbinary-reader)
This link shows me how to load total data.
I think when the setting, "keepDataInMemory", is true and "randomize" is false, then my data set is totally loaded well.
Thank you for your comment.

I tried this, but It does not work, because I use "ImageReader".
The setting is for "TextFormatReader".

Few days ago, I used the "TextFormatReader", but I switched reader type from text to image, because Image file format, such as "png", reduce data size by compressing.
My data consists of only ( 0 or 1 ), so if the data transform to image, then the data size is rapidly decreased.

If a data set does not contain double or float type data, and the data set have limited range, such as 0~255) then, I suggest switching the reader type.
If Text format dim = 1000000 then, image input size 1000*1000 (data size maybe reduced to 1/10 ~ 1/1000), save the loading time.
